BUILD.bazel

```diff
@@ -257,7 +258,7 @@ orfs_flow(
             "GPL_ROUTABILITY_DRIVEN": "1",
             "GPL_TIMING_DRIVEN": "0",
         },
-        "route": SKIP_REPORT_METRICS,
+	"route": SKIP_REPORT_METRICS | { "GENERATE_ARTIFACTS_ON_FAILURE": "1" },
```

Comment:
Practically, this won't do anything until we have a new version in ORFS that breaks out grt in a separate stage and it should be set by bazel-orfs.

Running a build now. I'll merge this sans this last line, if it works, as a new baseline.

Reply:
Hmmm... actually, these two PDN strategies go together as a pair. One for submacros and another for the top level, so further changes are needed...

```
$ grep -r PDN_TCL ../OpenROAD-flow-scripts/flow/designs/asap7/mock-array/
../OpenROAD-flow-scripts/flow/designs/asap7/mock-array/config.mk:  export PDN_TCL               = $(PLATFORM_DIR)/openRoad/pdn/BLOCKS_grid_strategy.tcl
../OpenROAD-flow-scripts/flow/designs/asap7/mock-array/Element/config.mk:export PDN_TCL                = $(FLOW_HOME)/platforms/asap7/openRoad/pdn/BLOCK_grid_strategy.tcl

```